spyderlib/spyder.py

```diff
@@ -942,6 +948,12 @@ def create_edit_action(text, tr_text, icon_name):
             support_action = create_action(self,
                                            _("Spyder support..."),
                                            triggered=self.google_group)
+            check_updates_trig = lambda: self.check_updates(True)
+            check_updates_action = create_action(self,

```

Comment:
I think you should use here `self.check_updates_action` directly and remove the line below.